schema/test/test.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const assert = require('assert');
+const Ajv2020 = require("ajv/dist/2020")
+const ajv = new Ajv2020()
+
+const verifiableCredentialSchema = "/../verifiable-credential-schema.json"
```

Comment:
This file has a mix of semi-colon / non-semi-colon lines ... it should be normalized to use semi-colons.

Reply:
thanks I try to avoid JS at all costs so consider me a noob